packages/terra-action-header/docs/README.md

```diff
@@ -7,6 +7,11 @@ The terra-action-header component is a header bar containing a title and optiona
 - Install with [npmjs](https://www.npmjs.com):
   - `npm install terra-action-header`
 
+## Implementation Notes:
+The Action-Header component must be composed inside the [Base][1] component with a locale in order for it to load the correct translation strings. These translation strings are used for the default Action-Header title when the application does not specify a title for the Action-Header.
```

Comment:
I think it adds maintainability to mention, "These translation strings are used for the default Action-Header title when the application does not specify a title for the Action-Header." If it is removed or if more text is internationalized in the future, then the developer has to come and update it here.